common/cryptfs.c

```diff
@@ -144,6 +151,77 @@ convert_key_to_hex_ascii(unsigned char *master_key, unsigned int keysize,
 }
 #endif
 
+static int
+load_integrity_mapping_table(const char *real_blk_name, const char *name, int fs_size)
+{
+	//general variables
+	int fd_mapper;
+	int ioctl_ret;
+
+	//load mapping variables
+	char mapping_buffer[DM_INTEGRITY_BUF_SIZE];
+	struct dm_target_spec *tgt;
+	struct dm_ioctl *mapping_io;
+	char *integrity_params;
+	char *extra_params = "0";
+	int mapping_counter;
+
+	//Load Integrity map table
+	// ioctl(fd_mapper, DM_TABLE_LOAD, io)
+	DEBUG("Loading Integrity mapping table");
+
+	if ((fd_mapper = open(DEV_MAPPER, O_RDWR)) < 0) {
+		ERROR("Cannot open device-mapper in load integrity mapping table");
```

Comment:
ERROR_ERRNO

Reply:
Adapted occurrences to use ERROR_ERRNO after syscalls and ERROR else